server/main.go

```diff
@@ -73,6 +73,8 @@ func init() {
 }
 
 func main() {
+	// set max number of CPUs, disable log printing
+	maxprocs.Set(maxprocs.Logger(nil))
```

Comment:
only for json?

Reply:
yes, Its log is not controlled by zap and will break the format of zap json log

Reply:
this line will be executed even if for console format